frontend/pages/policies/ManagePoliciesPage/components/OtherWorkflowsModal/OtherWorkflowsModal.tsx

```diff
@@ -405,28 +426,14 @@ const ManagePolicyAutomationsModal = ({
               </>
             )}
           </div>
-          <div className={`form-field ${baseClass}__workflow`}>
-            <div className="form-field__label">Workflow</div>
-            <Radio
-              className={`${baseClass}__radio-input`}
-              label="Ticket"
-              id="ticket-radio-btn"
-              checked={!isWebhookEnabled}
-              value="ticket"
-              name="ticket"
-              onChange={onChangeRadio}
+          <p className={`${baseClass}__info`}>
+            The workflow will be triggered when hosts fail these policies.{" "}
```

Comment:
This text should have the `help-text` styles – there's a `help-text` mixin if that's the easiest route